src/sql/inspect.py

```diff
@@ -220,7 +292,12 @@ def __init__(self, table_name, schema=None) -> None:
                                 """,
                 ).fetchall()
 
-                table_stats[column]["mean"] = float(results_avg[0][0])
+                if is_numeric:
+                    table_stats[column]["mean"] = format(
```

Comment:
There are multiple code blocks like :

```
if is_numeric:
 ...
else:
  ...
```

Can we consolidate these in a couple of functions - one for `is_numeric = true` other for False

Reply:
created a function `_assign_column_specific_stats` that takes in ` table_stats[column] ` and `is_numeric` and does what is done in this block : 
```
if is_numeric:
 ...
else:
  ...
```

Now we won't see those multiple blocks in code.